packages/lesswrong/lib/collections/comments/views.ts

```diff
@@ -503,11 +503,16 @@ ensureIndex(Comments,
   { name: "comments.tagId" }
 );
 
+// TODO merge with subforumFeedSortings
```

Comment:
I tried to do this (add sorting by relevance and inflation adjusted karma, and make either subforumFeedSortings or subforumSorting the source of truth), but it turned out to be fairly complicated. Mainly because the mixed type feed resolvers don't support  synthetic fields in an obvious way. This means that there are two extra sorting available for the posts list in the wiki tab.

This was also part of my reason for not removing the wiki tab posts list (yet), in case people want to use those sortings. The other reason is so that long time users can keep using the wiki page as they usually do. I don't love how the posts list is duplicated though

Reply:
Could you make it so that the sorting only appears for the list view.... ehhhhhhhh... that's pretty bad. I dunno, might be worth it, but I dunno, prolly not this PR.

Reply:
Yeah I tried that and I wasn't happy with it, also even that wasn't that straightforward because of the need to sort the discussions too